federation/pkg/kubefed/init/init.go

```diff
@@ -582,6 +601,42 @@ func createControllerManager(clientset *client.Clientset, namespace, name, svcNa
 	return clientset.Extensions().Deployments(namespace).Create(dep)
 }
 
+func waitForPods(clientset *client.Clientset, name, nameSpace string) error {
```

Comment:
Why not make `waitForPods()` take a slice of names `waitForPods(..., name []string, ...)` and check for both the pods together instead of having to call them separately? Pluralized name for the function is already suggesting that.

Also, convention is to call the variable `namespace` - no capitalization for `S`.